com.unity.UnityHub.appdata.xml

```diff
@@ -7,14 +7,15 @@
   <summary>Real-time game engine</summary>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>LicenseRef-proprietary</project_license>
-  <url type="homepage">https://unity3d.com/</url>
-  <description>
-    <p>Unity is used to create half of the world’s games. Our real time platform, powered by tools and services, offer incredible possibilities for game developers, and creators across industries and applications.</p>
-    <p>NOTE: This wrapper is not verified by, affiliated with, or supported by Unity Technologies.</p>
-  </description>
+  <url type="homepage">https://unity3d.com</url>
+  <url type="help">https://forum.unity.com</url>
+  <url type="bugtracker">https://github.com/flathub/com.unity.UnityHub/issues</url>
+  <description><p>Unity is used to create half of the world’s games. Our real time platform, powered by tools and services, offer incredible possibilities for game developers, and creators across industries and applications.</p><p>NOTE: This wrapper is not verified by, affiliated with, or supported by Unity Technologies.</p></description>
```

Comment:
Could we leave the second paragraph on a new line? Yeah it's kinda bikeshedding but makes it a bit easier to edit & visualize.